src/main/scala/firrtl/passes/CheckHighForm.scala

```diff
@@ -191,7 +191,6 @@ trait CheckHighFormLike { this: Pass =>
         case ex => ex foreach validSubexp(info, mname)
       }
       e foreach checkHighFormW(info, mname + "/" + e.serialize)
```

Comment:
```suggestion

```
Can you not drop this by the same logic? It doesn't do any meaningful work, but it would keep things kind of consistent.

Reply:
I originally thought the same thing, but literals actually do have user-defined types so we do need to check them.